hkmc2/shared/src/main/scala/hkmc2/codegen/Lifter.scala

```diff
@@ -99,7 +103,16 @@ object Lifter:
         case Some(value: BlockMemberSymbol) => S(value)
         case _ => N
       case _ => N
+  
+  object InstSel:
+    def unapply(p: Path) = p match
+      case Value.Ref(l: BlockMemberSymbol) => S(l)
+      case s @ Select(Value.Ref(l: BlockMemberSymbol), Tree.Ident("class")) => S(l)
+      case _ => N
 
+  def modOrObj(d: Defn) = d match
+    case c: ClsLikeDefn => (c.k is syntax.Mod) || (c.k is syntax.Obj)
+    case _ => false
```

Comment:
You're missing an empty line:
```suggestion
    case _ => false

```